docs/content/tutorial/resize-cluster.md

```diff
@@ -23,7 +23,8 @@ It's important for instances in the same high availability set to have the same
 - `spec.backups.pgbackrest.repoHost.resources` section, which sets the resources for the pgBackRest repo host container, as well as any init containers in the associated pod and containers created by the `pgBackRestVolume` [data migration job]({{< relref "guides/data-migration.md" >}}).
 - `spec.backups.pgbackrest.sidecars.pgbackrest.resources` section, which sets the resources for the `pgbackrest` sidecar container.
 - `spec.backups.pgbackrest.jobs.resources` section, which sets the resources for any pgBackRest backup job.
-- `spec.backups.pgbackrest.restore.resources` section, which sets the resources for any pgBackRest restore job.
+- `spec.backups.pgbackrest.restore.resources` section, which sets the resources for manual pgBackRest restore jobs.
+- `spec.dataSource.postgresCluster.resources` section, which sets the resources for pgBackRest restore jobs created during the [cloning]({{< relref "./disaster-recovery.md" >}}) process or when performing a [PITR]({{< relref "./disaster-recovery.md" >}}).
```

Comment:
:wrench: I'm thinking we can removed the `"or when performing a PITR"` part of the `spec.dataSource.postgresCluster.resources` description, since PITRs aren't specific to these "data source" restores (PITR's are applicable to manual restores as well, so seems a bit confusing to have them called out here, but not in `spec.backups.pgbackrest.restore.resources`).